source/hassio/haos_common_tasks.markdown

```diff
@@ -0,0 +1,140 @@
+---
+title: "Home Assistant OS Common Tasks"
+description: "Guides for common tasks when using Home Assistant OS"
+---
+
+
+## Configuring access to files
+
+One of the first things to take care of after installing Home Assistant OS is to provide yourself access to files. There are several add-ons commonly used for this, and most users employ a mix of a handful of add-ons. This section will describe some common choices.
+
+---
+
+### Installing and using the Samba add-on
+
+Your Home Assistant OS install has default directories which are mapped to the add-ons so that they can be accessed by the services a particular add-on might provide. The Samba add-on exposes all of these as smb shares which can then easily be accessed from another computer. You can also edit files using the editor of your preference from your client computer.
+
+The following directories are made available:
```

Comment:
Any reason this is part of the Samba add-on info and not part of the intro to accessing files ?